src/variables.py

```diff
@@ -10,6 +10,12 @@
 # - Account -
 ACCOUNT = None
 MEMBER_PRIV_KEY = os.getenv('MEMBER_PRIV_KEY')
+
+MEMBER_PRIV_KEY_FILE = os.getenv('MEMBER_PRIV_KEY_FILE')
+if MEMBER_PRIV_KEY_FILE is not None:
```

Comment:
```
if not os.path.exists(MEMBER_PRIV_KEY_FILE): 
    raise ValueError(f"File {MEMBER_PRIV_KEY_FILE} is not found")
```

Reply:
Yeah, let's add more details, like "provided path is not valid" or smth similar

Reply:
Ok, I'll add today that